Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js

```diff
@@ -276,75 +269,106 @@ define([
-                              'frustumStatisticText', 'pickTileActive', 'pickPrimitiveActive', 'hasPickedPrimitive',
-                              'hasPickedTile', 'tileText', 'generalVisible', 'generalSwitchText',
-                              'primitivesVisible', 'primitivesSwitchText', 'terrainVisible', 'terrainSwitchText', 'depthFrustumText']);
+        knockout.track(this, [
+            'frustums',
+            'performance',
+            'shaderCacheText',
+            'primitiveBoundingSphere',
+            'primitiveReferenceFrame',
+            'filterPrimitive',
+            'tileBoundingSphere',
+            'filterTile',
+            'wireframe',
+            'globeDepth',
+            'pickDepth',
+            'depthFrustum',
+            'suspendUpdates',
+            'tileCoordinates',
+            'frustumStatisticText',
+            'tileText',
+            'hasPickedPrimitive',
+            'hasPickedTile',
+            'pickPrimitiveActive',
+            'pickTileActive',
+            'dropDownVisible',
+            'generalVisible',
+            'primitivesVisible',
+            'terrainVisible',
+            'depthFrustumText'
+        ]);
 
         this._toggleDropDown = createCommand(function() {
             that.dropDownVisible = !that.dropDownVisible;
         });
 
         this._toggleGeneral = createCommand(function() {
             that.generalVisible = !that.generalVisible;
-            that.generalSwitchText = that.generalVisible ? '-' : '+';
         });
 
         this._togglePrimitives = createCommand(function() {
             that.primitivesVisible = !that.primitivesVisible;
-            that.primitivesSwitchText = that.primitivesVisible ? '-' : '+';
         });
 
         this._toggleTerrain = createCommand(function() {
             that.terrainVisible = !that.terrainVisible;
-            that.terrainSwitchText = that.terrainVisible ? '-' : '+';
         });
 
-        this._showFrustums = createCommand(function() {
-            if (that.frustums) {
-                that._scene.debugShowFrustums = true;
-            } else {
-                that._scene.debugShowFrustums = false;
-            }
-            return true;
+        this._frustumsSubscription = knockout.getObservable(this, 'frustums').subscribe(function(val) {
+            that._scene.debugShowFrustums = val;
         });
 
-        this._showPerformance = createCommand(function() {
-            if (that.performance) {
+        this._performanceSubscription = knockout.getObservable(this, 'performance').subscribe(function(val) {
+            if (val) {
                 that._performanceDisplay = new PerformanceDisplay({
                     container : that._performanceContainer
                 });
             } else {
                 that._performanceContainer.innerHTML = '';
             }
-            return true;
         });
 
         this._showPrimitiveBoundingSphere = createCommand(function() {
```

Comment:
Oh, nevermind.  I see it's being used in multiple places, just not as a knockout binding.